msyncd/AccountsHelper.cpp

```diff
@@ -42,7 +44,8 @@ AccountsHelper::AccountsHelper(ProfileManager &aProfileManager, QObject *aParent
     QObject::connect(iAccountManager, SIGNAL(accountUpdated(Accounts::AccountId)),
                      this, SLOT(slotAccountUpdated(Accounts::AccountId)));
 
-    registerAccountListeners();
+    // load accounts after return from contructor
+    QTimer::singleShot(0, this, SLOT(registerAccountListeners()));
```

Comment:
could just use QMetaObject::invokeMethod(this, "registerAccountListeners", Qt::QueuedConnection); I guess?